enatega-multivendor-rider/src/components/OrderDetail/Details/useDetails.js

```
@@ -79,58 +81,56 @@ const useDetails = orderData => {
-          o => o._id === result.updateOrderStatusRider._id
+        const index = data?.riderOrders?.findIndex(
+          o => o._id === result?.updateOrderStatusRider?._id   
         )
         if (index > -1) {
           data.riderOrders[index].orderStatus =
-            result.updateOrderStatusRider.orderStatus
+            result?.updateOrderStatusRider?.orderStatus
           cache.writeQuery({
             query: ORDERS,
-            data: { riderOrders: [...data.riderOrders] }
+            data: { riderOrders: [...data?.riderOrders] }
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)